web/packages/teleterm/src/ui/DocumentCluster/ClusterResources/Databases/Databases.tsx

```diff
@@ -57,10 +58,13 @@ function DatabaseList(props: State) {
   const dbs = fetchAttempt.data?.agentsList.map(makeDatabase) || [];
   const disabled = fetchAttempt.status === 'processing';
   const loggedInUser = useWorkspaceLoggedInUser();
+  const { clusterUri } = useClusterContext();
+  const canAddResources =
+    routing.isRootCluster(clusterUri) && loggedInUser?.acl?.tokens.create;
```

Comment:
This mirrors the logic in Web UI.

https://github.com/gravitational/teleport/blob/4d094a7743244e665266240099e0cde6346133ef/web/packages/teleport/src/components/AgentButtonAdd/AgentButtonAdd.tsx#L27